### PR TITLE
#165382765 include mail template in auth routes

### PR DESCRIPTION
**Pull request by Chiazokam**
#### Description
This PR includes the mail template in the sign up and login routes

#### Type of change
- [x] Non-breaking change

#### How Has This Been Tested?
image

image

#### Checklist:
- [x] include the mail template in the auth routes
- [x] modify the mail title

#### Story (PT)
[#165382765](https://www.pivotaltracker.com/story/show/165382765)

### CODE REVIEW

File: server/controllers/user.controller.js [hunk 1]
```diff
@@ -50,12 +50,14 @@ class UserController {
...
+    const body = `<p>Thanks for joining and we hope you read stories that change your life forever!</p>
+    Warm regards.`;
+
     const mailOption = {
       email,
       subject: 'Welcome to Author\'s Haven',
-      message: `<h3>Hi ${firstname},
-      <p>Thanks for joining and we hope you read stories that change your life forever!</p>
-      Warm regards.`,
+      message: mailTemplate(`<h3>Hello ${firstname} ${lastname},</h3>`, body)
```
Comment (chukwudidu) [inline server/controllers/user.controller.js]: Hi, @Chiazokam I think we can do without the `<h3>` tag here. It was not used in the `login` mail.

**Review by chukwudidu — CHANGES_REQUESTED** [review]

**Review by TheDrizzyWay — COMMENTED** [review]
This looks good.

**Review by TheDrizzyWay — APPROVED** [review]

**Review by geekdanitek — APPROVED** [review]

**Review by samorahno — APPROVED** [review]

**Comment by BukkyOmo** [comment]
> This looks good.

Looks good to me

File: server/controllers/user.controller.js (hunk 1 above)
Comment (Chiazokam) [reply server/controllers/user.controller.js]: Alright....will do this

Comment (Chiazokam) [reply server/controllers/user.controller.js]: Done

**Review by chukwudidu — APPROVED** [review]

**Review by davidokonji — APPROVED** [review]

**Review by paradisekelechi — APPROVED** [review]

**Pull request merged**